app/src/main/res/drawable/ic_short_hair_24dp.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="31.688"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_short_hair_24dp` appears to be unused
 Unused resources make applications larger and slow down builds.